pkg/openapiclient/local_crds.go

```diff
@@ -37,15 +38,18 @@ func (g inmemoryGroupVersion) Schema(contentType string) ([]byte, error) {
 
 // Dir should have openapi files following directory layout:
 // myCRD.yaml (groupversions read from file)
-func NewLocalCRDFiles(dirPath string) openapi.Client {
-	return &localCRDsClient{dir: dirPath}
+func NewLocalCRDFiles(fs fs.FS, dirPath string) openapi.Client {
```

Comment:
Could also you change the Patch client to use same args order? I think the order you've chosen makes more sense

https://github.com/kubernetes-sigs/kubectl-validate/blob/5f4d23caa3afe49a9c2b4298688436789e5538f2/pkg/openapiclient/overlay.go#L21

Reply:
Done